README.md

```diff
@@ -67,6 +67,21 @@ where:
 
 _This method calls `group.add` and `group.addAuthors` for you (adding you)_
 
+### `ssb.tribes.application.create(groupdId, text, cb)`
+
+Creates a tribe application message directed to the administrators of a private-group.
+
+### `ssb.tribes.application.get(applicationId, cb)`
+
+Returns the current state of a tribe application.
+
+### `ssb.tribes.application.list(groupId, accepted, cb)`
+
+Returns a list with all group applications. **groupId** and **accepted** are optional (pass `null` if not used), and can be used to filter applications for a specific group or applications that have been accepted or not.
+
+### `ssb.tribes.application.accept(applicationId, text, cb)`
+
+This runs `ssb.tribes.invite` to invite the person to the group, then publishes an update to the application linking to that message.
```

Comment:
With the APIs I've been creating I've been following a pattern something like : 

```
function (required_arg, { optional }, cb) 
```

the advantages of this are : 
- it's clear what's optional, because it's generally in an object
- the optional args are named (keys of object)
- it's easy to extend in the future - you can add more keys to opts object - which doesn't break the API

Reply:
If I have time  I'll modify these functions to follow that pattern

Reply:
Got it!